### PR TITLE
Feature/element missing broadcast

### PR DESCRIPTION
**Pull request by guidotripaldi**

### CODE REVIEW

File: lib/drab/element.ex [hunk 1]
```diff
@@ -288,6 +288,29 @@ defmodule Drab.Element do
...
+  Broadcasting version of `set_style/3`.
+
+  It does exactly the same as `set_style/3`, but instead of pushing the message to the current
+  browser, it broadcasts it to all connected users.
+
+  Always returns `{:ok, :broadcasted}`.
+
+  See `Drab.Core.broadcast_js/2` for broadcasting options.  
+  """
+  @spec broadcast_style(Drab.Core.subject(), String.t(), map | Keyword.t()) :: Drab.Core.result()
```
Comment (grych) [inline lib/drab/element.ex]: Return of broadcasting functions should be type of `Drab.Core.bcast_result()`

File: lib/drab/element.ex [hunk 2]
```diff
@@ -288,6 +288,29 @@ defmodule Drab.Element do
...
+  def broadcast_style(subject, selector, properties) when is_list(properties) or is_map(properties) do
+    broadcast_js(subject, set_js(selector, %{"style" => Map.new(properties)}))
+  end
+
+  @doc """
+  Exception-throwing version of `broadcast_style/3`
+  """
+  @spec broadcast_style!(Drab.Core.subject(), String.t(), map | Keyword.t()) :: Drab.Core.return() | no_return
```
Comment (grych) [inline lib/drab/element.ex]: There is no need to have bang versions of broadcasting functions, as they work in "fire and forget" regime.

File: lib/drab/element.ex (hunk 1 above)
Comment (guidotripaldi) [reply lib/drab/element.ex]: ah, yes! Damned copy/past..

File: lib/drab/element.ex (hunk 2 above)
Comment (guidotripaldi) [reply lib/drab/element.ex]: ok. I saw that the bang version for broadcast_js exist so I thinked it was better to add these versions

Comment (grych) [reply lib/drab/element.ex]: True, it should not be there, I am going to depreciate this.

**Pull request merged**